tests/unit/array/mixins/test_embed.py

```diff
@@ -111,3 +117,52 @@ def test_embedding_on_random_network(
         da[: int(N / 2)].embed(embed_model, batch_size=batch_size, to_numpy=to_numpy)
         da[-int(N / 2) :].embed(embed_model, batch_size=batch_size, to_numpy=to_numpy)
         np.testing.assert_array_almost_equal(da.embeddings, embed1)
+
+
+@pytest.fixture
+def paddle_model():
+    import paddle.nn as nn
+
+    class DummyPaddleLayer(nn.Layer):
+        def forward(self, x, y):
+            return (x + y) / 2.0
+
+    return DummyPaddleLayer()
+
+
+def test_embeded_paddle_model(paddle_model):
+    def collate_fn(da):
+        return {'x': da.tensors, 'y': da.tensors}
+
+    docs = DocumentArray.empty(3)
+    docs.tensors = np.random.random([3, 5]).astype(np.float32)
+    docs.embed(paddle_model, collate_fn=collate_fn, to_numpy=True)
+    assert (docs.tensors == docs.embeddings).all()
+    # assert np.testing.assert_array_almost_equal(docs.tensors, docs.embeddings)
```

Comment:
this needs to be uncommented?

Reply:
`np.testing.assert_array_almost_equal` cannot work here. I have no idea what's the problem.